maintainer-guide.md

```diff
@@ -118,11 +120,25 @@ New releases are made every set period of time (typically every week), in which
 
 **Role: PM**
 
-* Pull the latest `release` branch.
-* Deploy to the live server.
-* Get live green, or otherwise all test failures need to be accounted for.
-* Make the version default.
-* Close the "Release" issue.
+Release without schema change:
+
+1. Pull the latest `release` branch.
+2. Deploy to the live server.
+3. Get live green, or otherwise all test failures need to be accounted for.
+4. Make the version default.
+5. Close the "Release" issue.
+
+Release with schema change:
+
+1. Pull the latest `release` branch.
+2. Take system to maintenance mode.
```

Comment:
There's no need for this

Reply:
As schema change on production is not automatic, if PM run liquibase command to update schema then deploy without maintenance mode, there might be some temporary error in between this process  cc @NicolasCwy

Reply:
Yes I suggested this since, if we deploy the release there would be an inconsistency between the database schema needed by a file e.g an action and the actual state of the DB before running the liquibase command to alter / add the new table

Reply:
@NicolasCwy That if anything is a lapse in coding. Code should be compatible between the existing version and the immediate next version.